sdk/servicebus/service-bus/package.json

```diff
@@ -62,7 +62,7 @@
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"samples/**/*.{ts,js}\" \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 120000 --full-trace dist-esm/test/*.spec.js dist-esm/test/**/*.spec.js",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 120000 --full-trace dist-esm/test/*.spec.js dist-esm/test/internal/**/*.spec.js dist-esm/test/public/**/*.spec.js",
```

Comment:
Shouldn't the integration-test command run all the tests?

Reply:
It is running all tests. I have to specify the subfolders separately bcz glob patterns don't work on Linux :(

Reply:
No, by "all" - I meant unit tests too.

Reply:
was it running before?

Reply:
I didn't have a reason before.

Now that I think more about it, it makes sense to have all the tests being part of the integration-test command because we are providing the code coverage reports in DevOps only for the live tests and not unit-tests.